contracts/HybridCustody.cdc

```diff
@@ -245,6 +246,10 @@ pub contract HybridCustody {
         pub fun removeChild(addr: Address) {
             if let cap = self.accounts.remove(key: addr) {
                 // TODO: Add access(contract) methods that flow down to ChildAccount s.t. parent is removed if exists in ChildAccount.parents
```

Comment:
Nitpick: I believe this TODO refers to the changes in this PR. If so, we should remove it